build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+function src {
+	mkdir -p "$DIR"/build
+	(cd $DIR/build && cmake ..)
+	(cd "$DIR"/build && make -j8)
+
+	if [ $? -eq 0 ]; then
+    	printf "\n${GREEN}|*******| SOURCE COMPILATION SUCCESSFUL |*******|${NC}\n\n"
+	else
+    	printf "\n${RED}|*******| SOURCE COMPILATION FAILED |*******|${NC}\n\n"
+    	exit 1
+	fi
+
+	cp "$DIR"/build/src/TerminalFighter/TerminalFighter "$DIR"/TerminalFighter
+}
+
+function format {
+	(cd "$DIR" && astyle *.h *.cpp -r --options=astylerc)
+	find "$DIR" -name "*.orig" -delete
+}
+
+function docs {
+	(cd "$DIR"/documentation && doxygen -g)
+	(cd "$DIR"/documentation && patch --no-backup-if-mismatch < $DIR/documentation/Doxyfile.patch)
+	(cd "$DIR"/documentation && doxygen Doxyfile)
```

Comment:
replace this line with:
```
(doxygen "$DIR"/documentation/Doxyfile)
```
Doxygen has to be run on the root directory so it can find "./src" (defined in the doxyfile)

Tested this on both vanilla bash (non-docker), and with docker and it works.